core/src/config.rs

```diff
@@ -103,3 +103,23 @@ fn admin_public_key_from(
 
     Ok(PublicKey::from(&SecretKey::from_bytes(&hash)?))
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn admin_public_key() {
+        let email: String = "pj@aa.pl".to_string();
+        let password: String =  "password".to_string();
+        let seed: Option<[u8; 32]> = Some([55; 32]);
+        let expected_pubkey: [u8; 32] = [17, 243, 42, 222, 75, 47, 128, 87, 1, 252, 72, 56, 141, 216, 210, 251, 217, 95, 97, 62, 95, 112, 234, 31, 243, 73, 64, 160, 134, 92, 138, 97];
+
+        let (_config, _) = Config::new(email, password, seed).unwrap();
+        if let Config::V1{seed: _, admin} = _config {
```

Comment:
`_config` means it's not being used. It should be `config` instead.

Reply:
👍